packages/build-utils/src/prerender.ts

```diff
@@ -45,8 +45,8 @@ export class Prerender {
 
     this.lambda = lambda;
     if (this.lambda) {
-      // "ISR" is the platform default lambda label for prerender functions
-      this.lambda.operationType = this.lambda.operationType || 'ISR';
+      // "SSG" is the platform default lambda label for prerender functions
+      this.lambda.operationType = this.lambda.operationType || 'SSG';
```

Comment:
I don't think we should be labeling this as SSG as mentioned in slack since a Prerender implies ISR especially with On-Demand Revalidate now.